### ISSUE TITLE
Question about optimizing the own mesh.

### ISSUE THREAD

**Issue by yiboz2001** [WU-CVGL/MVControl-threestudio#13]
Hi, thanks for the open-sourced project!
I try to use my own mesh (without texture) as the initialization of stage 3 and hope to optimize the corresponding SuGaR using default vsd loss, but the texture of final result seems bad

![7](https://github.com/user-attachments/assets/0a38b480-346e-457b-895c-a3115da844ab)

For the loss term, I only use the "2d" guidance.
```   
        out_2d = self.training_substep(batch, batch_idx, guidance="2d")
        total_loss += out_2d["loss"]
        # if self.guidance_control is not None and self.C(self.cfg.loss.lambda_sds_control) > 0:
        #     out_mvcontrol = self.training_substep(batch, batch_idx, guidance="mvcontrol")
        #     total_loss += out_mvcontrol["loss"]
        # out_ref = self.training_substep(batch, batch_idx, guidance="ref")
        # total_loss += out_ref["loss"] 
```

I will highly appreciate it if you have any suggestion for improving the result.
